youtube_dl/extractor/threeplus.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from .threeqsdn import ThreeQSDNIE
+from ..utils import get_element_by_class
+
+
+class ThreePlusBaseIE(InfoExtractor):
+    _HOST_URL = 'http://playout.3qsdn.com/'
+
+    def _get_title_and_description(self, video_id):
+        webpage = self._download_webpage(
+            self._HOST_URL + video_id, video_id)
+        title = self._og_search_title(webpage)
+        description = self._og_search_description(webpage)
+        return title, description
+
+    def _get_real_video_id(self, video_id):
+        video_info = self._download_webpage(self._HOST_URL + video_id + '?js=true', video_id)
```

Comment:
You should add support for this playlist-alike 3qsdn URLs in any non-breaking way.